frontend/src/components/common/authorized-menu/navigation-menu/navigation-menu-item/navigation-menu-item.tsx

```diff
@@ -0,0 +1,42 @@
+import { AppRoute } from 'common/enums/enums';
+import { FC, IconName, PathMatch } from 'common/types/types';
+import { Icon, Link } from 'components/common/common';
+import { getValidClasses } from 'helpers/helpers';
+
+import styles from './styles.module.scss';
+
+interface Props {
```

Comment:
We use type for such cases
```suggestion
type Props {
```